### ISSUE TITLE
Troubleshooting for low volume

### ISSUE THREAD

**Issue by signaleleven** [hassio-addons/addon-spotify-connect#265]
# Problem/Motivation

All the other issues complaining about low-volume have been closed, so I had to open a new one for those that land here

## Expected behavior

The addon outputs a volume comparable to other sources (such as a MPD addon)

## Actual behavior

The output volume is very low even if set to 100% in spotify

## Steps to reproduce

I don't know how it came to be, but I have a solution that at least worked for me.

## Proposed changes

The maximum volume, at least in my case, was too low because the pulseaudio volume for spotify was too low.
From a terminal addon (assuming hassos) run the command `pulsemixer`

While spotify is playing, raise the volume of its pulseaudio session 
![image](https://github.com/user-attachments/assets/8a4b7852-a147-49a0-9c05-18a642981bb9)

This solved the problem for me, although I am not sure what lowered it in the first place.
Changing the volume in spotify does not alter that setting, it's only changing the volume within librespot (the two volumes are applied sequencially)

I hope this helps, the issue can be closed, but I wanted to leave a mark here because all the issues about this problem have gone stale and were closed.

**Comment by eeeeeeek** [comment]
Thanks a lot for bringing up a solution for my almost 2 years old issue #179 !
On my home assistant device I don't have `pulsemixer`, but instead I've entered `pactl list sink-inputs short` while running spotify to determine the index of its sink-input and adjusted the volume with `pactl set-sink-input-volume 7 0.5`

**Comment by paulsantek** [comment]
Thank you both for pointing me in the right direction. After trying to work around the skipping issue spotify played again, but no sound leaving the speakers ( I did try a ton of other plugins getting some audio played, one of them music assistant with some plugin to have it play locally, didn't work either - maybe I screwed something up with that).

I had to use pactl as pulsemixer is not installed on my HA-Raspberry.
